examples/blog/src/pages/index.astro

```diff
@@ -1,81 +1,56 @@
 ---
-// Component Imports
 import BaseHead from "../components/BaseHead.astro";
-import BlogHeader from "../components/BlogHeader.astro";
+import Header from "../components/Header.astro";
 import BlogPostPreview from "../components/BlogPostPreview.astro";
 
-// Component Script:
-// You can write any JavaScript/TypeScript that you'd like here.
-// It will run during the build, but never in the browser.
-// All variables are available to use in the HTML template below.
 let title = "Example Blog";
 let description = "The perfect starter for your perfect blog.";
-let permalink = "https://example.com/";
 
-// Data Fetching: List all Markdown posts in the repo.
-
-let allPosts = await Astro.glob("./posts/*.md");
-allPosts = allPosts.sort(
-	(a, b) =>
-		new Date(b.frontmatter.publishDate).valueOf() - new Date(a.frontmatter.publishDate).valueOf()
-);
-
-// Full Astro Component Syntax:
-// https://docs.astro.build/core-concepts/astro-components/
+// Use Astro.glob to fetch all post with associated frontmatter
+const posts = await Astro.glob("./posts/*.md");
```

Comment:
How come the sorting is not necessary anymore? "How to sort things" is a common question we get on the Discord, so it'd feel best to have it demonstrated in one of our examples imo

Reply:
Ah, good catch! That was accidentally lost in the refactor. Putting back :thu